src/main/resources/default/assets/scripts/list-layout-switch.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
```

Comment:
Nachdem der Javascript ja in tycho-biz.js zusammengemergt wird, würde ich den Copyright Kommentar hier rauslassen. Oder zumindest von der Methode drunter durch eine Leerzeile absetzen, damit es nicht wie eine Dokumentation davon aussieht.